sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1,2 +1,350 @@
+that can be handled by `sem create`: secrets and projects configuration files.
+
+### sem delete
+
+The `sem delete` command is used for deleting existing resources, which means
+that is used for deleting Semaphore 2.0 projects and entire `secrets` buckets.
+
+Note that when you delete a `secrets` bucket, then that particular `secrets`
+bucket disappears from the active organization for every user that is a member
+of that organization and there is no way to get it back unless you recreate
+that `secrets` buckets using the `sem create` command along with the
+appropriate YAML file. However, when you use `sem delete` to delete a project,
+what happens is that that particular project is deleted from the active
+organization but all of its files along with the related GitHub repository
+remain intact.
+
+### sem describe
+
+The `sem describe` command returns the description of an existing resource.
+The resource type as well as the resource name should be given as command line
+arguments, in that order.
+
+### sem get
+
+The `sem get` command returns the list of items for the given resource type
+that can be found in the active organization and requires a command line
+argument, which is the resource type.
+
+## Project Initialization
+
+This group only includes the `sem init` command.
+
+### sem init
+
+The `sem init` command adds a GitHub repository as a Semaphore 2.0 project to
+the active organization.
+
+The `sem init` command should be executed from within the root directory of a
+GitHub repository that has been either created locally and pushed to GitHub or
+cloned using the `git clone` command. The command is executed without any other
+command line parameters or arguments.
+
+## Help commands
+
+The last group of `sem` commands includes the `sem help` and `sem version`
+commands, which are help commands.
+
+### sem help
+
+The `sem help` command returns information about an existing command when
+it is followed by a valid command name. If no command is given as a
+command line argument to `sem help`, a help screen is printed.
+
+### sem version
+
+The `sem version` command requires no additional command line parameters and
+returns the current version of the `sem` tool.
+
+### Resources
+
+There exist two kinds of resources: `secrets` bucket and projects. Each
+resource is uniquely identified by its name.
+
+#### Secrets
+
+You can consider a `secrets` bucket as a place where you can store your
+sensitive data.
+
+Each `secrets` bucket is associated with a single organization. In other words,
+a `secrets` bucket belongs to an organization. In order to use a specific
+`secrets` bucket you should be connected to its organization.
+
+Currently, there is no way to use two or more `secrets` buckets that belong in
+different organizations in the same Semaphore 2.0 project. You should import
+the desired `secrets` buckets in a single organization and used them from
+there.
```

Comment:
Remove this. There is no need to be defensive with this paragraph about something that no one actually expects to be possible.

Reply:
👍